client/src/auth/index.tsx

```diff
@@ -2,41 +2,40 @@ import React from "react";
 
 import { PageContentContainer } from "../ui/atoms/page-content";
 
+import "./index.scss";
```

Comment:
I don't like this. 
The main CSS bundle is super precious and we have to be super vigilant about any bloat. 

I think you should simply remove this line here and inject it into the `sign-in.tsx` and the `sign-up.tsx` and I think that'll work.